pkg/network/http/compile.go

```diff
@@ -22,6 +24,13 @@ func getRuntimeCompiledHTTP(config *config.Config) (runtime.CompiledOutput, erro
 
 func getCFlags(config *config.Config) []string {
 	var cflags []string
+
+	if strings.ToLower(rt.GOARCH) == "arm64" {
```

Comment:
may just fixup only arch that need to
```
arch := strings.ToLower(rt.GOARCH)
if arch ==  "arm64" {
  arch = "aarch64"
}
cflags = append(cflags, fmt.Sprintf("--target=%s-unknown-linux-gnu", arch))
```

Reply:
I think the `strings.ToLower` is unnecessary. Here are all possible values: https://github.com/golang/go/blob/master/src/go/build/syslist.go#L11

Reply:
We should check for `arm64` and `amd64` here if we want to be explicit and error out in out in other cases. Bit overkill, but I think more future proof.